test_schemas/en/test_routing_case_insensitive_text_field.json

```diff
@@ -0,0 +1,112 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.2",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Routing is case insensitive for text fields",
+    "theme": "default",
+    "description": "A questionnaire to test case insensitive routing on text fields",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "country-text-field",
+                            "question": {
+                                "id": "country-text-field-question",
+                                "title": "Name a country you have visited",
+                                "type": "General",
```

Comment:
In some schemas we have guidance to say what to type to test routing, ie with dates. I think we should do the same as I wouldn't know what to type to test this without looking at the source.

Reply:
Like these: https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/test_schemas/en/test_routing_date_equals.json#L45-L60